RecoLocalCalo/HGCalRecProducers/plugins/HGCalMultiClusterProducer.cc

```diff
@@ -29,101 +28,98 @@
 
-HGCalMultiClusterProducer::HGCalMultiClusterProducer(const edm::ParameterSet &ps) :
-  doSharing(ps.getParameter<bool>("doSharing")),
-  verbosity((HGCalClusteringAlgoBase::VerbosityLevel)ps.getUntrackedParameter<unsigned int>("verbosity",3)){
-  std::vector<double> multicluster_radii = ps.getParameter<std::vector<double> >("multiclusterRadii");
+HGCalMultiClusterProducer::HGCalMultiClusterProducer(const edm::ParameterSet& ps)
+    : doSharing(ps.getParameter<bool>("doSharing")),
+      verbosity((HGCalClusteringAlgoBase::VerbosityLevel)ps.getUntrackedParameter<unsigned int>("verbosity", 3)) {
+  std::vector<double> multicluster_radii = ps.getParameter<std::vector<double>>("multiclusterRadii");
   double minClusters = ps.getParameter<unsigned>("minClusters");
-  clusters_token = consumes<std::vector<reco::BasicCluster> >(ps.getParameter<edm::InputTag>("HGCLayerClusters"));
-  clusters_sharing_token = consumes<std::vector<reco::BasicCluster> >(ps.getParameter<edm::InputTag>("HGCLayerClustersSharing"));
+  clusters_token = consumes<std::vector<reco::BasicCluster>>(ps.getParameter<edm::InputTag>("HGCLayerClusters"));
+  clusters_sharing_token =
+      consumes<std::vector<reco::BasicCluster>>(ps.getParameter<edm::InputTag>("HGCLayerClustersSharing"));
   hits_ee_token = consumes<HGCRecHitCollection>(ps.getParameter<edm::InputTag>("HGCEEInput"));
   hits_fh_token = consumes<HGCRecHitCollection>(ps.getParameter<edm::InputTag>("HGCFHInput"));
   hits_bh_token = consumes<HGCRecHitCollection>(ps.getParameter<edm::InputTag>("HGCBHInput"));
   auto sumes = consumesCollector();
 
   multicluster_algo = std::make_unique<HGCal3DClustering>(ps, sumes, multicluster_radii, minClusters);
 
-  produces<std::vector<reco::HGCalMultiCluster> >();
-  produces<std::vector<reco::HGCalMultiCluster> >("sharing");
+  produces<std::vector<reco::HGCalMultiCluster>>();
+  produces<std::vector<reco::HGCalMultiCluster>>("sharing");
 }
 
-
 void HGCalMultiClusterProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   // hgcalMultiClusters
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("HGCLayerClusters", edm::InputTag("hgcalLayerClusters"));
   desc.addUntracked<unsigned int>("verbosity", 3);
   desc.add<bool>("doSharing", false);
-  desc.add<edm::InputTag>("HGCEEInput", edm::InputTag("HGCalRecHit","HGCEERecHits"));
-  desc.add<edm::InputTag>("HGCFHInput", edm::InputTag("HGCalRecHit","HGCHEFRecHits"));
-  desc.add<std::vector<double>>("multiclusterRadii", {
-    2.0,
-    5.0,
-    5.0,
-  });
-  desc.add<edm::InputTag>("HGCBHInput", edm::InputTag("HGCalRecHit","HGCHEBRecHits"));
-  desc.add<edm::InputTag>("HGCLayerClustersSharing", edm::InputTag("hgcalLayerClusters","sharing"));
+  desc.add<edm::InputTag>("HGCEEInput", edm::InputTag("HGCalRecHit", "HGCEERecHits"));
+  desc.add<edm::InputTag>("HGCFHInput", edm::InputTag("HGCalRecHit", "HGCHEFRecHits"));
+  desc.add<std::vector<double>>("multiclusterRadii",
```

Comment:
Maybe, having it in a single line would be better...
Not due to the clang formatter, though, as it was as such also in the original code